discord-scripts/role-management.ts

```diff
@@ -0,0 +1,34 @@
+import { Client } from "discord.js"
+
+export default async function manageRole(
```

Comment:
I would add this as an unexported helper to the renamed `role-management` script from #293 . If we do move to a directory for role management due to complexity, this can end up in a `helpers` file or similar.